go/oasis-node/cmd/config/migrate/migrate.go

```diff
@@ -297,16 +308,16 @@ func doMigrateConfig(cmd *cobra.Command, args []string) {
 						mkSubMap(m(newCfg["consensus"]), "p2p")
 						for pk, pv := range m(v) {
 							if pk == "persistent_peer" {
-								m(m(newCfg["consensus"])["p2p"])["persistent_peers"] = convertP2P(pv, "consensus.tendermint.p2p.persistent_peer", "consensus.p2p.persistent_peers")
+								m(m(newCfg["consensus"])["p2p"])["persistent_peers"] = convertP2P(pv, "consensus.tendermint.p2p.persistent_peer", "consensus.p2p.persistent_peers", false)
 								delete(m(m(tendermint)["p2p"]), pk)
 								continue
 							} else if pk == "unconditional_peer" || pk == "unconditional_peer_ids" {
-								m(m(newCfg["consensus"])["p2p"])["unconditional_peers"] = convertP2P(pv, fmt.Sprintf("consensus.tendermint.p2p.%s", pk), "consensus.p2p.unconditional_peers")
+								m(m(newCfg["consensus"])["p2p"])["unconditional_peers"] = convertP2P(pv, fmt.Sprintf("consensus.tendermint.p2p.%s", pk), "consensus.p2p.unconditional_peers", false)
 								delete(m(m(tendermint)["p2p"]), pk)
 								continue
 							} else if pk == "seed" {
 								mkSubMap(newCfg, "p2p")
-								m(newCfg["p2p"])["seeds"] = convertP2P(pv, "consensus.tendermint.p2p.seed", "p2p.seeds")
+								m(newCfg["p2p"])["seeds"] = convertP2P(pv, "consensus.tendermint.p2p.seed", "p2p.seeds", true)
```

Comment:
```suggestion
								m(newCfg["p2p"])["seeds"] = appendLibp2pAddresses(convertP2P(pv, "consensus.tendermint.p2p.seed", "p2p.seeds"))
```
I would probably move the code to another function (e.g. `appendLibp2pAddresses`, `appendSeedP2P`, ...), as the description of the `translateKnownP2P` disagrees with changes. You also get rid of the `isSeed` flag.